ChemistryLib/CreateChemicalSolverInterface.cpp

```diff
@@ -186,9 +187,15 @@ createChemicalSolverInterface<ChemicalSolver::PhreeqcKernel>(
         //! \ogs_file_param{prj__chemical_system__rates}
         config.getConfigSubtreeOptional("rates"));
 
+    // equilibrium reactants
+    auto equilibrium_reactants = PhreeqcKernelData::createEquilibriumReactants(
+        //! \ogs_file_param{prj__chemical_system__equilibrium_phases}
+        config.getConfigSubtreeOptional("equilibrium_phases"), mesh);
```

Comment:
Just a question. equilibrium phases are same as equilibrium reactants?

Reply:
Absolutely.

Reply:
Then I would suggest to use only one name, if possible. For me the equality is not obvious.

I understand, that this is one more rename through whole code. Maybe in a discussion a choice could be made before any actual renames...

Reply:
How about update the tag in the .prj file?

Reply:
For me both options are fine, (reactants or phases) but I agree with @endJunction that homogeneous nomenclature facilitate things, maybe to make it clear that we are doing "reactive" things I would suggest "equilibrium_reactants"

Reply:
I start to update the tag as suggested.

Reply:
What we define in the .prj file is `phase components`. So the data structure in the .prj file would look like either of the below
```
<equilibrium_reactants>
    <equilibrium_reactant>
        ...
    </equilibrium_reactant>
</equilibrium_reactants>
```

```
<equilibrium_reactants>
    <phase_component>
        ...
    <phase_component>
</equilibrium_reactants>
```
which data structure will we use?

Reply:
I like the first one.

Reply:
Note that all the dataset between the tag `<equilibrium_reactant>` and `</equilibrium_reactant>` will go into the built-in object `std::vector<PhaseComponent> phase_components`.

Reply:
I see, you have to stop somewhere. One step at a time, but keep the big picture in your head.
If we want this consistent on all layers, then there are few renames to do. Can be independent.
And also every thrive for more consistency will almost always break current semi-consistency; this is unavoidable.

For this PR I'd maybe stop at the `phase_component` as in your second option above.

Reply:
Let's take this compromise plan at the present. When we have better workaround to the naming issue, we polish it again.

Reply:
Done including Docu, tag replacement in prj files, renaming class member variables and functions in `ChemistryLib::PhreeqcIO` and `ChemistryLib::PhreeqcKernel`.